src/templates/Plugin.js

```diff
@@ -66,7 +66,7 @@ const PluginTemplate = ({ data }) => {
           {plugin.frontmatter.gettingStarted && (
             <ResponsiveSpacer>
               <div className="text-center pb-3" >
-                <Title text="Installation steps" />
+                <Title text="Self-hosted Backstage installation steps" />
```

Comment:
I'm trying to make it more clear that these are instructions for self-hosted Backstage. Not for Roadie users.